integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPortForward.java

```diff
@@ -0,0 +1,443 @@
+    administrationPortDomainNamespace = namespaces.get(2);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace,
+        adminDefaultPortDomainNamespace, administrationPortDomainNamespace);
+
+    // create and verify WebLogic domain image for adminDefaultPortDomain
+    // using model in image with model files
+    String imageName = createAndVerifyDomainImage(adminDefaultPortDomainNamespace,
+        PORTFORWARD_ADMIN_DEFAULT_IMAGE_NAME, PORTFORWARD_ADMIN_DEFAULT_MODEL_FILE);
+
+    // create and verify the domain is created and ready to use
+    logger.info("Create domain and verify that it's running");
+    createAndVerifyDomain(imageName, adminDefaultPortDomainNamespace, adminDefaultPortDomainUid);
+
+    // create and verify WebLogic domain image for administrationPortDomain
+    // using model in image with model files
+    imageName = createAndVerifyDomainImage(administrationPortDomainNamespace,
+        PORTFORWARD_ADMINISTRATION_IMAGE_NAME, PORTFORWARD_ADMINISTRATION_MODEL_FILE);
+
+    // create and verify the domain is created and ready to use
+    createAndVerifyDomain(imageName, administrationPortDomainNamespace, administrationPortDomainUid);
+  }
+
+  /**
+   * kill port-forward process.
+   */
+  @AfterAll
+  void tearDown() {
+    stopPortForwardProcess();
+  }
+
+  /**
+   * The test that `kubectl port-foward` is able to forward a local port to default channel port
+   * (7001 in this test).
+   * Verify that the WLS admin console can be accessed using http://localhost:localPort/console/login/LoginForm.jsp
+   */
+  @Order(1)
+  @Test
+  @DisplayName("Forward a local port to default channel port and verify WLS admin console is accessible")
+  @DisabledIfEnvironmentVariable(named = "OKD", matches = "true")
+  void testPortForwardDefaultAdminChannel() {
+    final int adminDefaultChannelPort = 7001;
+
+    // Verify that using a local port admin forwarded to admin's default channel (7001),
+    // admin console is accessible via http://localhost:localPort/console/login/LoginForm.jsp
+    String portForwardFileName = portForwardFileNameProfix + "-1.out";
+    startPortForwardProcess(adminDefaultPortDomainNamespace,
+        adminDefaultPortDomainUid, adminDefaultChannelPort, portForwardFileName);
+    verifyAdminConsoleAccessible(adminDefaultPortDomainNamespace, portForwardFileName, false);
+  }
+
+  /**
+   * The test that `kubectl port-foward` is able to forward a local port to default secure channel port
+   * (7002 in this test).
+   * Verify that the WLS admin console can be accessed using http://localhost:localPort/console/login/LoginForm.jsp
+   */
+  @Order(2)
+  @Test
+  @DisplayName("Forward a local port to default secure channel port and verify WLS admin console is accessible")
+  @DisabledIfEnvironmentVariable(named = "OKD", matches = "true")
+  void testPortForwardDefaultAdminSecureChannel() {
+    final int adminDefaultChannelPort = 7002;
+
```

Comment:
Description is already in the method level javdoc.

Reply:
done